auction-app-frontend/src/App.jsx

```diff
@@ -21,10 +29,12 @@ const App = () => {
   return (
     <>
       <BreadcrumbProvider>
+      <SuggestionProvider>
         <Header />
         <Navbar />
         <div className="main-content">
           <div className="container">
+            <SuggestionBox />
```

Comment:
Can suggestion box be part of the Navbar
As I remember, this is where your search filed is 
You can simplify logic by putting it there.